core/micro-services/workflow-core/src/main/protobuf/edu/uci/ics/amber/workflowruntimestate.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
```

Comment:
add the generated scala files.

Reply:
Seems that all generated scala files are generated after `micro-service/workflow-core` is loaded locally, they are under `core/micro-services/workflow-core/target/scala-2.13/src_managed`, ignored by .gitignore

Reply:
no scalapb will generate it under `src/main/scalapb` directory. Can you check again?

Reply:
under `micro-services/workflow-core/src/main` I only see 
- `protobuf` for pb definition
- `resources`
- `scala` for amber core codes

Also in this commit: https://github.com/Texera/texera/commit/78b0a49f4cab31505aeb592cde95ccf053551346
I don't see the generated scala files.

Reply:
got it. lets follow the new practice: not commit protobuf generated code for scalapb.